File: backend/member-service/src/main/java/com/example/memberservice/profile/dto/ProfileDto.java
package com.example.memberservice.profile.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProfileDto implements Serializable {

    private Long id;
    private Long memberId;
    private ProfileType profileType;
    private String value;
    private String createdDate;
    private String updatedDate;
}

File: android/modu-chat/app/src/main/kotlin/com/example/modumessenger/data/dto/ProfileDto.kt
package com.example.modumessenger.data.dto

data class ProfileDto(
    val id: Long? = null,
    val memberId: Long? = null,
    /** `PROFILE_STATUS_MESSAGE` / `PROFILE_WALLPAPER` / `PROFILE_IMAGE`. */
    val profileType: String? = null,
    val value: String? = null,
    val createdDate: String? = null,
    val updatedDate: String? = null,
)

data class CreateProfileDto(
    val memberId: Long,
    val profileType: String,
    val value: String,
)
